CONTRIBUTING.md

```diff
@@ -76,19 +76,44 @@ These will be taken care of as part of the setup git commands, but for reference
+
+   First, download the most recent version of Nucleus Coop from [Releases](https://github.com/SplitScreen-Me/splitscreenme-nucleus/releases). Next you will want to compile Nucleus Coop itself following the directions in the following section. Last, copy and paste the following files from the root folder of the release build download's root into the root directory of your new Nucleus Coop build (the "Release" or "Debug" subfolder contained within "\splitscreenme-nucleus\Master\NucleusCoopTool\bin\"):
+     * ProtoInputHooks32.dll
+     * ProtoInputHooks64.dll
+     * ProtoInputHost.exe
+     * ProtoInputIJ32.exe
+     * ProtoInputIJ64.exe
+     * ProtoInputIJP32.dll
+     * ProtoInputIJP64.dll
+     * ProtoInputLoader32.dll
+     * ProtoInputLoader64.dll
+     * ProtoInputUtilDynamic32.dll
+     * ProtoInputUtilDynamic64.dll
+
+   This ensures that the files that would have been built by compiling Proto Input are placed in the correct location in the Nucleus Coop build folder with the executable.
+   
+   **Option 2:** Batch Build **Proto Input** by opening its solution file in Visual Studio and batch building it ("Submodules\ProtoInput\src\ProtoInput\ProtoInput.sln" within the project root).
+  
+     The cleanest way to do this is probably to sort by "Solution Config" header under the Batch Build menu and check the boxes that correspond with the version you would like to compile (make sure to check both x86 and x64 boxes for your selected build type).
+
+     Check out the troubleshooting section for additional tips on getting Proto Input to compile correctly, if you have any problems! Proto Input appears to be very sensitive to your development environment, so you will likely need to consult this section, especially if you are trying to work with it within Visual Studio 2022.
+
+     Close the Proto Input project out when you're done.
 
 You are now free to work on the Nucleus Coop source code - all dependencies in place! If you get any errors in compiling along the way, you may need to install and run project solution files from an earlier version of Visual Studio or the .NET SDK for better project compatibility.
 
 ### Compiling and Testing Changes
 
-When you are ready to test any changes you make, compile **Nucleus Coop** by opening its solution file and running a batch build on it within Visual Studio ("Master\NucleusCoop.sln"). Similar to Proto Input, on the Batch Build menu, select all the x86 and x64 options for the build you would like to compile. 
+When you are ready to test any changes you make, compile **Nucleus Coop** by opening its solution file and running a batch build on it within Visual Studio ("Master\NucleusCoop.sln"). 
+
+**Make sure that your project path contains no folders with spaces in their name, or Nucleus Coop may not compile correctly.**

```

Comment:
I was really torn on this one... but I like how "important" flags it without triggering my "something is broken" response like the bright red caution one does when I read it. I'm leaning towards a follow-up commit that blends some of the suggested phrasing in the "caution" suggestion with it too, because I think the stronger tone is a bit clearer (though I think I'll slightly edit it to "Your project path _should_ not contain..."